core/templates/pages/exploration-player-page/templates/lesson-information-card-modal.component.spec.ts

```diff
@@ -436,4 +436,39 @@ describe('Lesson Information card modal component', () => {
 
     expect(componentInstance.saveProgressMenuIsShown).toBeFalse();
   });
+
+  it('should return 0% when no checkpoints are completed', () => {
+    componentInstance.completedCheckpointsCount = 0;
+    componentInstance.checkpointCount = 5;
```

Comment:
Add a new line after this line. Please do the same for the remaining tests

Reply:
done.